src/sql/workbench/contrib/charts/browser/graphInsight.ts

```diff
@@ -159,28 +163,33 @@ export class Graph implements IInsight {
 			return mixin(c, getColors(this.options.type, i, c.data!.length), false);
 		});
 
+		if (this.options.type === 'horizontalBar') {
+			this.options.type = ChartType.Bar;
+			this.options.indexAxis = 'y';
+		}
 		if (this.chartjs) {
 			this.chartjs.data.datasets = chartData;
-			this.chartjs.config.type = this.options.type;
+			(<chartjs.ChartConfiguration>this.chartjs.config).type = this.options.type as chartjs.ChartType;
```

Comment:
Instead of using as (which is risky since you're bypassing the type system) you should have a mapping helper function to map our internal ChartType to the chartjs one.